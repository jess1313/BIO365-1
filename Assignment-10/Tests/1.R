suppressPackageStartupMessages(suppressWarnings(suppressMessages(source("1.R"))))

numVector1 = 1:50
numVector2 = rep(c(1:5), 50)
set.seed(1)
numVector3 = rnorm(1000)

print(round(calcMean(numVector1), 8))
print(round(calcMean(numVector2), 8))
print(round(calcMean(numVector3), 8))