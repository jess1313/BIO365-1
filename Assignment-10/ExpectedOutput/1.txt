[1] 25.5
[1] 3
[1] -0.01164814
